src/index.tsx

```diff
@@ -10,8 +10,9 @@ import { ROLES } from './Roles';
  * @param {createjs.Stage} stage - createjs stage that has been registered for accessibility
  * @returns {Object} style object for positioning associated DOM elements
  */
-function calcDomStylesFromStage(stage) {
-  // true to return style object for putting the DOM element adjacent to the canvas (useful for debugging),
+function calcDomStylesFromStage(stage): { [key: string]: React.CSSProperties } {
```

Comment:
- not familiar with the return type here, curious as to how you came up with it

Reply:
So the above function returns css styles as object. Defining styles type as shown, will preserve type check on all CSS classes and their properties.